change/flamegrill-2019-10-10-22-20-33-master.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "minor",
```

Comment:
This can be `patch` since the API didn't change. 😄

Reply:
haha absolutely fair

Reply:
Sorry for all the churn. You could have edited this file instead of making a new one. Either way is fine, but this file should be deleted to prevent an automatic minor rev bump when this PR gets merged.

Reply:
ah no worries